packages/fxa-admin-panel/package.json

```diff
@@ -53,7 +53,7 @@
     "serve-static": "^1.16.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.26.0",
+    "@babel/core": "7.25.7",
```

Comment:
Downgrading?

Reply:
I think it's best to keep package groups aligned to the same version, especially if we intend to eventually move these down to root to make future maintenance easier.

The @babel group had a range of versions between 7.16 and 7.26.  I couldn't find any @babel/core resolutions in the 7.26 range, so I chose 7.25.7 which seemed the most widely used in the group and still relatively recent (released Oct).

I'm happy to try to bump everything up to 7.26 – it's possible some adjacent packages may not go that high, or issues arise (which we could try to resolve via code/test fixes).  I time-boxed myself on this PR since it ballooned from the original ask, but I'm happy to keep going on it!

Reply:
>  I'm happy to try to bump everything up to 7.26

No, or at least I don't know enough about the babel packages to say.  But keeping them in lock-step sounds good.